### PR TITLE
[Feature] Export docs in multiple ways

### PR DESCRIPTION
**Pull request by brapoprod**
This PR adds all controllers to documentation
Also exports documentation to md files as well as html files
Updates the github pages deployment script
Updates the TS (and other dependencies) and typedoc versions
Removes docs from repo, but moved to buildtime

## Related tickets

-   [WRS-739](https://support.chili-publish.com/projects/WRS/issues/WRS-739)

### CODE REVIEW

**Review by abdelhalimkhouas — APPROVED** [review]

File: src/README_OLD.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+window.SDK.layout.addLayout(2);
+```
+
+<br/>
+
+### Recap
+
+So to recap, you first need to initialise the SDK and Editor, then add an empty div with the ID that you want to pass through the SDK to inject the Editor in.
+Then you're ready to go, if you initialise some listeners with callbacks or use a getter you can easily retrieve data on change or on request.
+To send data you just call the sdk method, with or without parameters (depends on the method).
```
Comment (brapoprod) [inline src/README_OLD.md:203]: I want to leave this for now, since it can come in handy in further documentation steps

File: src/README.md [hunk 2]
```diff
@@ -1,203 +1,20 @@
...
 
-Happy coding and enjoy the **spicy** ride.
+So on these pages you'll find some very technical but super handy descriptions of all the functions available in the controllers, as well as the exported enums and other configuration params.
 
-## General Structure
+## Controllers
 
-The SDK consist of controllers which are individual Javascript classes and methods of these classes.
-Currently there are 7 different controllers:
+-   [AnimationController](classes/controllers_AnimationController.AnimationController.md)
```
Comment (alexandraFlavia9) [inline src/README.md:9]: some controllers are missing

File: src/README_OLD.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+If you have any questions, do not hesitate to reach out to any of the chili team-members.
+
+Happy coding and enjoy the **spicy** ride.
+
+## General Structure
+
+The SDK consist of controllers which are individual Javascript classes and methods of these classes.
+Currently there are 7 different controllers:
+
+-   AnimationController
```
Comment (alexandraFlavia9) [inline src/README_OLD.md:14]: same as above

File: src/README_OLD.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+The SDK should also be aware that there is now room in the html for injecting the editor, so you should call the `loadEditor` method in your integration script.
+
+Example in `integration.js`:
+
+```javascript
+// at the bottom
+window.SDK.loadEditor();
+```
+
+If you now refresh your page in the browser you will see And like magic an instance of the editor gets loaded on your very own page.
```
Comment (alexandraFlavia9) [inline src/README_OLD.md:147]: something seems wrong in the wording

Comment (brapoprod) [reply src/README_OLD.md:147]: old readme, not used anymore but is there still as a reference, should be deleted soon

File: src/README_OLD.md (hunk 3 above)
Comment (brapoprod) [reply src/README_OLD.md:14]: old readme, not used anymore but is there still as a reference, should be deleted soon

File: src/README_OLD.md (hunk 1 above)
Comment (brapoprod) [reply src/README_OLD.md:203]: Done

File: src/README.md (hunk 2 above)
Comment (brapoprod) [reply src/README.md:9]: Done

**Review by alexandraFlavia9 — CHANGES_REQUESTED** [review]

File: src/README.md [hunk 5]
```diff
@@ -1,203 +1,25 @@
...
+-   [DocumentController](classes/controllers_DocumentController.DocumentController.md)
+-   [FrameController](classes/controllers_FrameController.FrameController.md)
+-   [LayoutController](classes/controllers_LayoutController.LayoutController.md)
+-   [ToolController](classes/controllers_ToolController.ToolController.md)
+-   [UtilsController](classes/controllers_UtilsController.UtilsController.md)
+-   [VariableController](classes/controllers_VariableController.VariableController.md)
+-   [DebugController](classes/controllers_DebugController.DebugController.md)
+-   [SubscriberController](classes/controllers_SubscriberController.SubscriberController.md)
+-   [PageController](classes/controllers_PageController.PageController.md)
+-   [SubscriberController](classes/controllers_SubscriberController.SubscriberController.md)
```
[the remark this answers is not in the document]
Comment (brapoprod) [reply src/README.md]: Done

File: src/README.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ So on these pages you'll find some very technical but super handy descriptions o
 -   [DebugController](classes/controllers_DebugController.DebugController.md)
 -   [SubscriberController](classes/controllers_SubscriberController.SubscriberController.md)
 -   [PageController](classes/controllers_PageController.PageController.md)
--   [SubscriberController](classes/controllers_SubscriberController.SubscriberController.md)
+-   [TextStyleController](classes/controllers_TextStyleController.TextStyleController.md)
 -   [UndoManagerController](classes/controllers_UndoManagerController.UndoManagerController.md)
 -   [Default methods (SDK)](classes/index.SDK.md)
```
Comment (alexandraFlavia9) [inline src/README.md:21]: and ColorStyleController

**Review by alexandraFlavia9 — CHANGES_REQUESTED** [review]

File: src/README.md (hunk 6 above)
Comment (brapoprod) [reply src/README.md:22]: Done

**Review by alexandraFlavia9 — APPROVED** [review]

**Pull request merged**